PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -89,8 +89,36 @@ public static Question[] LoadQuestions(string filePath)
                 question.Answers[1] = answer2;
                 question.Answers[2] = answer3;
                 question.CorrectAnswerIndex = correctAnswerIndex;
+                RandomizeAnswerOrder(question);
+                questions[i] = question;
             }
             return questions;
         }
+
+        public static void RandomizeAnswerOrder(Question question)
+        {
+            Random random = new Random();
+
+            for (int i = 0; i < question.Answers.Length; i++)
+            {
+                int randomAnswer = random.Next(0, 3);
+
+                if (randomAnswer != i)
+                {
+                    string tmp = question.Answers[i];
+                    question.Answers[i] = question.Answers[randomAnswer];
+                    question.Answers[randomAnswer] = tmp;
+
+                    if (Int32.Parse(question.CorrectAnswerIndex) == i + 1)
```

Comment:
nit: In general you want to favor C# keywords over type BCL type names. So in this case use `int` rather than `Int32`. The difference is subtle, if someone were to declare an Int32 type with a Parse method that was "closer", the compiler could resolve to it by mistake. Using the keywords avoids that risk, so where keywords exists we favor using them.
```suggestion
                    if (Int32.Parse(question.CorrectAnswerIndex) == i + 1)
```
```suggestion
                    if (int.Parse(question.CorrectAnswerIndex) == i + 1)
```